Color/Colors.php

```diff
@@ -167,150 +167,162 @@ class Colors
      *
      * @return array
      */
-    public static function getAll()
+    public static function getAll($isInverted = true)
```

Comment:
Might be a good idea, `Colors` does not seem to be used on its own. Or maybe just move the data (because hey, this class is basically just data) to a YML file (or any other suitable format), and load that file in the type?